utils/wasi-nn/download-pytorch-fixtures.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+DOWNLOAD_TO=$1
+FIXTURE=https://github.com/gusye1234/torchscript_fixtures/raw/main/mobilenet
```

Comment:
Is there a official site of the mobilenet pytorch model?

Reply:
Probably not. I didn't find any place that stores the torch-scripted models.